docs/user/supported_devices.rst

```diff
@@ -407,6 +407,7 @@ ramips-mt76x8
   - Archer C50 (v3)
   - Archer C50 (v4)
   - RE200 (v2)
+  - RE200 (v4)
```

Comment:
```suggestion
```

broken devices are not supported

Reply:
I can close this PR and reopen it by the time its fixed?

Reply:
You don't need to close this. You just need to remove it from the docs.
Once the issue is resolved you can open a PR that removes the `broken` tag and adds it as supported to the docs.